preupg/xmlgen/compose.py

```diff
@@ -306,15 +305,17 @@ def indent(cls, elem, level=0):
 
     @classmethod
     def get_template_file(cls):
-        return os.path.join(os.path.dirname(__file__), "template.xml")
+        return os.path.join(settings.data_dir, "templates",
+                            settings.xccdf_template)
 
     @classmethod
     def get_xml_tree(cls):
         template_file = ComposeXML.get_template_file()
         try:
             target_tree = ElementTree.parse(template_file).getroot()
         except IOError:
-            print('Problem with reading template.xml file')
+            print('Problem with reading {0} file'
```

Comment:
I can see it's used a lot in the other code. However, % instead of format is used also a lot.
But to be backwards compatible with python 2.4, I'll switch this to %.